assets/javascripts/discourse/components/canned-reply.js.es6

```diff
@@ -6,11 +6,6 @@ import computed from "ember-addons/ember-computed-decorators";
 export default Ember.Component.extend({
   isOpen: false,
 
-  @computed("isOpen")
```

Comment:
Why was this removed?

Reply:
`showContentIcon` was not used anywhere. It was removed in 4eb75be5e885d01a31cb2237db124d437c25cced.